lib/dor/services/client/mutate.rb

```diff
@@ -29,22 +29,23 @@ def apply_admin_policy_defaults
         # @param [Cocina::Models::DROWithMetadata|CollectionWithMetadata|AdminPolicyWithMetadata|DRO|Collection|AdminPolicy] params model object
         # @param [boolean] skip_lock do not provide ETag
         # @param [boolean] validate validate the response object
-        # @param [Hash] event_data additional event data to be added to the update object event
+        # @param [string] who the sunetid of the user performing the update
+        # @param [string] description a description of the update
         # @raise [NotFoundResponse] when the response is a 404 (object not found)
         # @raise [UnexpectedResponse] when the response is not successful.
         # @raise [BadRequestError] when ETag not provided.
         # @return [Cocina::Models::DROWithMetadata,Cocina::Models::CollectionWithMetadata,Cocina::Models::AdminPolicyWithMetadata] the returned model
         # rubocop:disable Metrics/AbcSize
         # rubocop:disable Metrics/MethodLength
-        def update(params:, skip_lock: false, validate: false, event_data: {})
+        def update(params:, skip_lock: false, validate: false, who: nil, description: nil)
           raise ArgumentError, 'Cocina object not provided.' unless params.respond_to?(:externalIdentifier)
 
           # Raised if Cocina::Models::*WithMetadata not provided.
           raise ArgumentError, 'ETag not provided.' unless skip_lock || params.respond_to?(:lock)
 
           resp = connection.patch do |req|
             req.url object_path
-            req.params = { event_data: event_data.to_json }
+            req.params = { description: description, who: who }.compact
```

Comment:
switch general event data as json into specific params needed to add to the event